handbook/quality/README.md

```diff
@@ -0,0 +1,146 @@
+
+### Weekly bug review
+QA has weekly check-in with product to go over the inbox items. QA is responsible for proposing “not a bug”, closing due to lack of response (with a nice message), or raising other relevant questions. All requires product agreement
+
+Requesters have six weeks to provide follow-up information for each info request. We will ping them again as a reminder at three weeks. After six weeks, we will close the bug to remove it from our visibility, but requesters are welcome to re-open and provide context.
+
+QA may also propose that a reported bug is not actually a bug. A bug is defined as “behavior which is not according to spec or implied by spec.” Thereafter, it is assigned to the relevant product manager for them to decide on its priority.
+
+### Acknowledging bugs
+Otherwise, QA should apply the acknowledged state to the bug. QA has one week to reproduce the bug.
+
+Once reproduced, QA should document the reproduction steps and move it to the reproduced state.
+
+### Reproduced
+When reproduced, the assigned engineering manager is responsible for investigating root cause of the bug and proposing solutions to their product counterpart if it requires discussion. Otherwise, the EM includes it in this release if there is space or the next release.
+
+### After reproduced
+After it is in a release formally, the bug should be treated like any other piece of work per the standard engineering process.
+
+### Fast track for Fleeties
+Fleeties do not have to wait for QA to reproduce the bug. If you are confident it is reproducible, is a bug, and the reproduction steps are well-documented, it can be moved directly to the reproduced state.
+
+### During release testing
+When release is in testing, QA should use the the slack channel #help-release to keep everyone aware of issues found. All bugs related to a release should be reported in the channel after creating the bug first.
+
+In the release channel, product may decide whether the bug is a release blocker. Release blockers must be fixed before a release can be cut.
+
+### Critical bugs
+A critical bug is defined as: “a bug that causes users to be unable to use a workflow, upgrade Fleet, or causes irreversible damage such as loss of data.”
+
+The key thing about a critical bug is that we need to immediately inform customers and the community about it so they don’t trigger it themselves. When bug meeting the definition of critical is found, the bug finder is responsible for raising an alarm immediately.
+Raising an alarm means: pinging @here in the #help-product channel with the filed bug.
+
+If the “bug finder” is not a Fleetie (such as community-reported), then whoever sees the critical bug should raise the alarm. (We would expect this to be CX in the community slack or QA in the bug inbox, though it could be anyone.)
```

Comment:
```suggestion
If the “bug finder” is not a Fleetie (such as community-reported), then whoever sees the critical bug should raise the alarm. (We would expect this to be CX in the community Slack or QA in the bug inbox, though it could be anyone.)
```